packages/ra-ui-materialui/src/list/ListView.tsx

```diff
@@ -70,7 +66,7 @@ export const ListView = <RecordType extends RaRecord = any>(
                       })
                     : children}
             </Content>
-            {pagination !== false && pagination}
+            {error ? <PaginationLimit /> : pagination !== false && pagination}
```

Comment:
I don't think you should say 'no results found' in case of error.

Reply:
Should I add an equivalent component to display an error message?